src/commands/moderation/LockCommand.js

```diff
@@ -0,0 +1,113 @@
+const Command = require('../../Command');
+const Discord = require('discord.js');
+const util = require('../../util');
+const ChannelConfig = require('../../ChannelConfig');
+
+const PERMS = ['SEND_MESSAGES', 'ADD_REACTIONS']
+
+class LockCommand extends Command {
+
+    static description = 'Lock channels (stop users from sending messages or add reactions)';
+
+    static usage = 'global|<#channel>…|<id>… <reason>';
+
+    static names = ['lock'];
+
+    static modCommand = true;
+
+    static userPerms = ['BAN_MEMBERS'];
```

Comment:
The old implementation required MANAGE_CHANNELS instead of BAN_MEMBERS (which makes more sense)
Is this intentional?

Reply:
Yes, that makes more sense